apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -104,12 +105,14 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
     const { link: { role } } = useOrganization()
 
     const action = async (variables, ...args) => {
+        const { details, ...otherVariables } = variables
         let createdContact
         if (role.canManageContacts && canCreateContactRef.current) {
             createdContact = await createContact(organization.id, selectPropertyIdRef.current, selectedUnitNameRef.current)
         }
         const result = await _action({
-            ...variables,
+            ...otherVariables,
+            details: normalizeText(details),
```

Comment:
could you please also normalize it on the server side. (use resolve input)

Reply:
done